src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,72 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    public static final String CSV_SEPARATOR = ",";
+    public static final int OPERATION_INDEX = 0;
+    public static final int VALUE_INDEX = 1;
+    public static final String BUY_NANE = "buy";
+    public static final String SUPPLY_NANE = "supply";
+    public static final String RESULT_NANE = "result";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        writeStatisticToFile(readStatisticFromFile(fromFileName), toFileName);
```

Comment:
```suggestion
        String dataFromFile = readFromFile(fromFileName);
        String report = createReport(dataFromFile);
        writeToFile(report, toFileName);
```
lets divide logic like this
first read data from file and then parse it

Reply:
Do you propose that I should change the whole MR? How can I work with string if I work with Object "Statistic"? I'm afraid I have to disagree with this suggestion. My logic is split. The first step is creating a 'Statistic' object from the file, and then I write the 'Statistic' to the file.